enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -499,7 +517,7 @@ def _create_redemption(
                 All other exceptions raised during the creation of an enrollment.  This should have already triggered
                 the rollback of a pending transaction.
         """
-        quantity = -1 * self.price_for_content(content_key)
+        quantity = -1 * content_price
```

Comment:
IDK why we made a second call to get the price here - we already have it from calling `is_redeemable()` further up in the body of `redeem()`.